generic-ec-core/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "generic-ec-core"
-version = "0.1.0"
+version = "0.1.2"
```

Comment:
Why are this and generic-ec bumped to 1.2?

Reply:
because I managed to fix the issue only after second attempt

Reply:
you can see the git history

Reply:
I see, it's because the versions were published to crates.io. I would vote for bumping all versions to 1.2 then

Reply:
Why bumping what has not been changed? If they were supposed to have the same version number, they would be a single lib

Reply:
If I could reupload library with the same version I'd do this, but it's impossible. Yanking doesn't delete crate version, it just marks it "do not use me", which is not even mandatory.

Reply:
I don't see strong arguments in favor of bumping all 4 crates, so I leave them as they are